src/js/components/ClusterPicker.js

```diff
@@ -25,13 +29,37 @@ const ClusterPickerWrapper = (styled.div`
 `: ComponentType<*>)
 
 export default function ClusterPicker() {
-  const current = useSelector(Current.mustGetConnection)
+  const dispatch = useDispatch()
+  const current = useSelector(Current.getConnection)
+  const clusters = useSelector(Clusters.all)
+
+  const template = clusters.map((c) => {
+    const isCurrent = c.id === current.id
+    return {
+      type: "checkbox",
+      label: c.id,
+      checked: isCurrent,
+      click: () => !isCurrent && dispatch(setConnection(c))
+    }
+  })
+
+  template.push(
+    {type: "separator"},
+    {
+      label: "+ New Connection",
+      click: () => dispatch(Current.setConnectionId(""))
```

Comment:
setConnectionId now accepts `null` for when there is no connection. I'd prefer that over the empty string we used to have.